### ISSUE TITLE
Spack create crashes due to undefined variable

### ISSUE THREAD

**Issue by adamjstewart** [LLNL/spack#407]
Here is the error message I get when trying to create a parallel-netcdf package:

```bash
$ spack create http://cucis.ece.northwestern.edu/projects/PnetCDF/Release/parallel-netcdf-1.6.1.tar.gz
==> This looks like a URL for parallel-netcdf version 1.6.1.
==> Creating template for package parallel-netcdf
Traceback (most recent call last):
  File "/soft/spack/bin/spack", line 176, in <module>
    main()
  File "/soft/spack/bin/spack", line 154, in main
    return_val = command(parser, args)
  File "/blues/gpfs/home/software/spack/lib/spack/spack/cmd/create.py", line 259, in create
    versions, urls = fetch_tarballs(url, name, args)
  File "/blues/gpfs/home/software/spack/lib/spack/spack/cmd/create.py", line 225, in fetch_tarballs
    versions = { version : url }
NameError: global name 'version' is not defined
```

**Comment by trws** [comment]
This appears to have been fixed.  Do you still see the problem @adamjstewart?

**Comment by adamjstewart** [comment]
I do still see the problem. I just downloaded the latest development branch on a different computer and tried to `spack create` the same url and I still get the same exact error message. The create.py file hasn't been edited since I posted this issue so I don't see how it could've been fixed. Are you able to `spack create` with this specific url? There's nothing wrong with the path because I can wget it fine. Maybe there is something else going on here...
